examples/taskrunner/app.js

```diff
@@ -27,14 +27,14 @@ function Task ({DOM, props, deleteComplete$, filter$}) {
 
   const delete$ = xs.merge(deleteClick$, deleteIfComplete$);
 
-  const viewUnlessFiltered  = (props, filter) =>
+  const viewUnlessFiltered  = ([props, filter]) =>
     taskView({
       ...props,
       visible: filter(props.status)
     });
 
   return {
-    DOM: xs.combine(viewUnlessFiltered, props, filter$),
+    DOM: xs.combine(props, filter$).map(viewUnlessFiltered),
```

Comment:
Any particular reason for the additional map here instead of passing viewUnlessFiltered as the first argument to combine?